tests/integration/db/test_sql_data_sampling.py

```diff
@@ -69,35 +54,54 @@ def _load_data(
         connection_string=connection_string,
         convert_colnames_to_datetime=["pickup_datetime", "dropoff_datetime"],
         load_full_dataset=True,
-        random_table_suffix=True,
+        random_table_suffix=random_table_suffix,
         to_sql_method=to_sql_method,
     )
 
 
+def _is_dialect_athena(dialect: str) -> bool:
+    """Is the dialect awsathena?"""
+    return dialect == "awsathena"
+
+
 if __name__ == "test_script_module":
 
-    dialect, connection_string = _get_connection_string_and_dialect()
+    dialect, connection_string = get_connection_string_and_dialect(
+        athena_db_name_env_var="ATHENA_TEN_TRIPS_DB_NAME"
+    )
     print(f"Testing dialect: {dialect}")
 
-    print("Preemptively cleaning old tables")
-    clean_up_tables_with_prefix(
-        connection_string=connection_string, table_prefix=f"{TAXI_DATA_TABLE_NAME}_"
-    )
+    if _is_dialect_athena(dialect):
+        athena_db_name: str = get_awsathena_db_name(
+            db_name_env_var="ATHENA_TEN_TRIPS_DB_NAME"
+        )
+        table_name: str = "ten_trips_from_each_month"
+        test_df: pd.DataFrame = load_and_concatenate_csvs(
+            csv_paths=[
+                f"./data/ten_trips_from_each_month/yellow_tripdata_sample_10_trips_from_each_month.csv"
+            ],
+            convert_column_names_to_datetime=["pickup_datetime", "dropoff_datetime"],
+            load_full_dataset=True,
+        )
+    else:
+        print("Preemptively cleaning old tables")
```

Comment:
pros/cons of making this a log statement? same throughout these tests.

Reply:
Hmm good question. I was following some other tests but glad you are pointing this out. Probably logging is better so we can set verbosity. I'm not sure how these messages show up in a) local runs b) in azure, would be good to test before making changes and make sure it meets our needs.